examples/Client/DistributedLockApi/TryLockApplication/Program.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Dapr.Client;
+
+namespace DistributedLockApi
+{
+    public class Program
+    {
+        [Obsolete]
+        public static async Task Main(string[] args)
```

Comment:
This feels strange to me as a sample because it intentionally exercises error cases, instead of demonstrating the correct way to do error handling. 

We should write a sample that shows how to use the API properly.

Reply:
I will be reworking on the examples to make them real world.